### DISCUSSION TITLE
No signal on Live video

### DISCUSSION THREAD

**Discussion by dushan90** [awawa-dev/HyperHDR#819] [Q&A]
I've installed HyperHDR on a RPi4 and using an Elgato HD60X as the capture card.

![image](https://github.com/awawa-dev/HyperHDR/assets/55048399/d37325cb-8dd8-467d-adb7-58adcb5592fa)

Even though the name of the capture card is visible in the device list, Live video says No signal and the lights aren't coming on either.

![image](https://github.com/awawa-dev/HyperHDR/assets/55048399/c0ff0f4d-f965-46ef-8f3d-69f4028d89de)

I've noticed that sometimes the lights will turn on when I change the device resolution. But that is like only for a second. During that one second I can see the video on Live video as well. I have also noticed that the lower the resolution, longer the lights stays on (lights turn on and immediately goes off if the resolution is set to 3840x2160 and will stay on for about 10 seconds if the resolution is set to 640x480)

Here is the log:
https://pastebin.com/gab6QHYG

**Comment by awawa-dev** [comment]
It's probably your device problem for example: insufficient power supply or low quality/broken USB-C cable or USB hub if connected as bridge.
When you receive no signal in HyperHDR preview window then ssh to RPi and search `dmesg` command output for errors.

**Reply by dushan90** [reply]
I'm using all the original cables came with the capture card and use a USB Y cable to supply power separately to the capture card.

Here is the `dmesg` command output
https://pastebin.com/thdRs919

**Reply by awawa-dev** [reply]
The stock cable is known to be faulty: not sure if it's a common problem or just some batch.
Read this topic: https://github.com/awawa-dev/HyperHDR/discussions/654#discussioncomment-7342611

>  USB Y cable to supply power separately to the capture card

Just another potential source of problems. Using the original power supply, RPi4 is able to provide power to Elgato itself and does not need such "helpers".

**Reply by dushan90** [reply]
I'll try to find a different cable and see it it works then. Thank you so much!
